src/Documentation/sidebar.json

```diff
@@ -119,6 +119,29 @@
         "label": "Managing External Data",
         "slug": "managing-external-data"
       },
+      {
+        "label": "Data Sharing",
+        "slug": "data-sharing",
+        "source": "data-sharing/index.md",
+        "children": [
+          {
+            "label": "Remote DVC Storage",
+            "slug": "remote-storage"
+          },
+          {
+            "label": "Shared Development Server",
```

Comment:
I'm not sure this one fits into this section. From the technical perspective it's the same as mounted DVC storage, right?

Reply:
in general last three look superficial, too specific. We should try and better generalize them (it's all about where cache is located, whether it is shared via `dvc cache dir` and whether remote is used/needed). It would be impossible to writer an article per each case and it's not needed. I would try to generalize this properly.

Reply:
> From the technical perspective it's the same as mounted DVC storage, right?

It seems to me that it is not the same as "mounted DVC storage", because in the case of mounted storage the project and the DVC storage are located on different file systems, so it can't be as efficient as the case of "shared development server".

The difference is subtle but the consequences are considerable (especially when we work with big data).

Reply:
> in general last three look superficial, too specific.

The first one seems the same to me as the last three (in terms of being detailed or superficial).

> We should try and better generalize them (it's all about where cache is located, whether it is shared via `dvc cache dir` and whether remote is used/needed). It would be impossible to writer an article per each case and it's not needed. I would try to generalize this properly.

I agree that the differences between them are subtle, but I think that they should be discussed as separate cases. At least I don't see how to generalize and merge them into a single page.